src/commands/game_options/groups.ts

```diff
@@ -7,10 +7,11 @@ import MessageContext from "../../structures/message_context";
 import { setIntersection } from "../../helpers/utils";
 
 const logger = new IPCLogger("groups");
+export const GROUP_LIST_URL = `http://${process.env.WEB_SERVER_IP}:${process.env.WEB_SERVER_PORT}/groups`;
 export default class GroupsCommand implements BaseCommand {
     help = {
         name: "groups",
-        description: `Select as many groups that you would like to hear from, separated by commas. A list of group names can be found [here](http://${process.env.WEB_SERVER_IP}:${process.env.WEB_SERVER_PORT}/groups).`,
+        description: `Select as many groups that you would like to hear from, separated by commas. A list of group names can be found [here](${GROUP_LIST_URL}).`,
```

Comment:
Maybe we should remove the reference to the full list of groups in the description? I'm unsure